lm_eval/tasks/exams/exams_openbook/_generate_configs.py

```diff
@@ -0,0 +1,59 @@
+"""
+Take in a YAML, and output all other splits with this YAML
+"""
+import argparse
+import os
+
+import requests
+import yaml
+from tqdm import tqdm
+
+from lm_eval.utils import logging
+
+
+API_URL = "https://datasets-server.huggingface.co/splits?dataset=mhardalov/exams"
+CONFIGS_TO_IGNORE = ['alignments', 'multilingual', 'multilingual_with_para']
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--base_yaml_path", required=True)
+    parser.add_argument("--save_prefix_path", default="sib200")
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    args = parse_args()
+
+    # get filename of base_yaml so we can `"include": ` it in our other YAMLs.
+    base_yaml_name = os.path.split(args.base_yaml_path)[-1]
+    with open(args.base_yaml_path, encoding="utf-8") as f:
+        base_yaml = yaml.full_load(f)
+
+    def query():
+        response = requests.get(API_URL)
+        return response.json()["splits"]
+    
+    languages = []
+    for lang_config in query():
+        if lang_config['config'] not in CONFIGS_TO_IGNORE and lang_config['split'] == 'validation' and 'para' in lang_config['config']:
+            languages.append(lang_config["config"])
+    
+    for lang in tqdm([lang for lang in languages]):
+        yaml_dict = {
+            "include": base_yaml_name,
+            "task": f"exams_open_book_{lang}",
+            "test_split": 'validation',
+            "fewshot_split": 'validation',
+            "dataset_name": f"{lang}"
+        }
+
+        file_save_path = args.save_prefix_path + f"_{lang}.yaml"
```

Comment:
nit: use `os.path.join` in case `args.save_prefix_path` is missing a trailing `/`

Reply:
I agree seems like bad coding practice, but it seems like the rest of the repo uses this method. I believe it is so that you can add a common name in the save prefix path without making it a subdirectory. For example you could specify /scratch/zoltanc/eval_2/lm-evaluation-harness/lm_eval/tasks/flores200/flores200 and then it would append flores200 infront of all the file names.

I lean toward keeping the configurations used in the rest of the tasks ie belebele, mmlu, csatqa etc..

Reply:
ah I see what you mean. sounds good then!